OneCommunityGlobal/HighestGoodNetworkApp
# Luis error message on admin page

# Description
(PRIORITY Low)  Jae/Joyce: Error message should disappear when search for a popupEditor item (WIP Luis) 
If a person goes to the admin page, enters “123”, and hits the search icon. The error message will appear in the consol when one clicks on the inspector.
<img width="550" alt="Screenshot 2023-08-14 at 10 15 57 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/22454953/a3769ace-83dc-4990-b4a0-60ee94784020">



## Main changes explained:
Was asked to fix the error message. When searching into the search bar. 
Figured out the error was because useState was setting data to null, so when searching it would throw an error. 
After speaking with Jae, he concluded it wasn't necessary since we can search via the browsers built in search function. 
See comments for details. 

<img width="490" alt="Screenshot 2023-08-14 at 8 35 15 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/22454953/f5fc3f24-71b6-4344-adee-b40c49e16a57">



## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as an admin user
4. go to dashboard→ other links → popup management.  
5. Search bar was removed and the error message is no longer there. 


## Screenshots or videos of changes:


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/22454953/c97bc5b5-cc32-41e4-baba-0fedd04d28c8
